src/sinks/elasticsearch.rs

```diff
@@ -130,13 +130,17 @@ impl ElasticSearchCommon {
                 if region.is_none() {
                     return Err(ParseError::AWSRequiresRegion.into());
                 }
-                Some(
-                    DefaultCredentialsProvider::new()
-                        .context(AWSCredentialsProviderFailed)?
-                        .credentials()
-                        .wait()
```

Comment:
I'm pretty sure we should just swap out `wait()` for `sync()` here.

Reply:
Not available with the https://docs.rs/rusoto_credential/0.41.1/rusoto_credential/trait.ProvideAwsCredentials.html trait.

Reply:
Oh. Weird that they're not using `RusotoFuture`. Carry on.